app/model/Cashbook/ReadModel/QueryHandlers/CategoryListQueryHandler.php

```diff
@@ -36,25 +31,15 @@ public function __construct(ICashbookRepository $cashbooks, CategoryRepository $
      */
     public function __invoke(CategoryListQuery $query) : array
     {
-        $cashbook = $this->cashbooks->find($query->getCashbookId());
-
+        $cashbook   = $this->cashbooks->find($query->getCashbookId());
         $categories = $this->categories->findForCashbook($cashbook->getId(), $cashbook->getType());
 
-        $calculator = new CategoryTotalsCalculator();
-
-        $totalByCategories = $calculator->calculate($cashbook, $categories);
-
-        // filter out camp refund categories
-        $categories = array_filter($categories, function (ICategory $category) {
-            return ! in_array($category->getId(), [ICategory::CATEGORY_REFUND_CHILD_ID, ICategory::CATEGORY_REFUND_ADULT_ID]);
-        });
-
         $categoriesById = [];
         foreach ($categories as $category) {
             $categoriesById[$category->getId()] = new Category(
                 $category->getId(),
                 $category->getName(),
-                MoneyFactory::fromFloat($totalByCategories[$category->getId()] ?? 0),
```

Comment:
Tomuhle bych se radši vyvaroval (nullovatelným fieldům, které ne/jsou null podle použité query) - pojďme mít dvě různý DTO. Jedno pro `CategoryListQueryHandler` a druhé pro ty součty (tam bude potřeba i méně dat)

Reply:
jasně, upravil jsem a přejmenoval